KBQA/kbqa/webservice/appA/app/nspm/interpreter_pt_model.py

```diff
@@ -141,15 +170,15 @@ def process_question(question):
     args = parser.parse_args()
     args.inputstr = question
     inputs = args.inputstr
-    # args.input = 'data/art_30'
-    args.input = "data/qald8"
-    args.ouput = "data/qald8"
-    # args.airml = 'http://nspm.org/art'
+    args.input = 'data/qald8'
+    # args.input = None
+    args.ouput = 'data/qald8'
+    args.airml = None
     # args.airml = None
     # model_dir = None
     # input_dir = None
-    model_dir = None
-    input_dir = None
+    model_dir = 'data/qald8'
+    input_dir = 'data/qald8'
     if args.input is not None:
```

Comment:
the lines about args are not necessary anymore, since we don't execute it from terminal